docs/components/IconDocs.js

```diff
@@ -80,6 +81,9 @@ class IconDocs extends React.Component {
           />
         </div>
 
+        <h5>elementProps <label>Object</label></h5>
+        <p>Pass props directly to the <Code>icon</Code> element. ie. focusable, etc.</p>
```

Comment:
I'm not sure how we worded this for other components that have `elementProps` but this seems a bit general and slightly miss leading as to the intent of elementProps.

This seems to be more in line with the spirit of what elementProps is.

`Pass native HTML attributes for SVG elements directly to the icon's wrapping SVG element.  ie. height, width, etc.`

Reply:
thanks, I appreciate this more thorough description. updated!